src/core/dataProviders/authProvider.js

```diff
@@ -35,13 +35,28 @@ const FetchLogin = (resolve, reject) => {
         });
 };
 
+const FetchLogout = (resolve, reject) => {
```

Comment:
I find it weird to extract the Promise callback like that. It makes your code harder to follow IMO.

Why do you need an additional Promise when you can write it all inline line 51:

```js
if (localStorage.getItem('userId') && token) {
    fetch(`${domainName}/api/logout`, options)
        .then(res => res.json())
        .then(response => {
            document.cookie = 'JSESSIONID=;';
            localStorage.removeItem('userId');
            localStorage.removeItem('username');
            localStorage.removeItem('role');
            window.location = response.redirectUrl;
         });
```

Reply:
Fixed. I removed promise ad changes code by this way.